CHANGELOG.rst

```diff
@@ -7,6 +7,29 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 1.7.3 (2024-06-04)
+--------------------------
+
+* Fixed: ensure correct data types
+  in dependency table
+* Fixed: publishing an update of a database
+  when the previous version
+  was stored in cache
+  by an older version of ``audb``
+* Fixed: loading of database attachments
+  when ``audb.config.CACHE_ROOT``
+  and ``audb.config.SHARED_CACHE_ROOT``
+  point to the same folder
+* Fixed: ensure ``audb.versions()``
+  is not failing
```

Comment:
I would normal present tense: "does not fail"

Reply:
Thanks, I changed accordingly.